pawn/Driver/Driver.pwn

```diff
@@ -224,12 +259,90 @@ public OnPlayerKeyStateChange(playerid, newkeys, oldkeys) {
             if (VehicleModel(modelId)->isNitroInjectionAvailable())
                 AddVehicleComponent(vehicleId, 1010);
         }
+
+        new bool: rightBlinker = g_blinkerObjects[playerid][0] != 0;
+        new bool: leftBlinker = g_blinkerObjects[playerid][2] != 0;
+        // Vehicle keys (q): blinker left
+        if (PRESSED(VEHICLE_KEYS_BINDING_BLINKER_RIGHT) && 
+            (vehicleKeys & VEHICLE_KEYS_BLINKER_RIGHT)) {
+
+            rightBlinker = !rightBlinker;
+        }
+
+        // Vehicle keys (e): blinker right
+        if (PRESSED(VEHICLE_KEYS_BINDING_BLINKER_LEFT) && 
+            (vehicleKeys & VEHICLE_KEYS_BLINKER_LEFT)) {
+
+            leftBlinker = !leftBlinker;
+        }
+
+        SetBlinker(playerid, vehicleId, leftBlinker, rightBlinker);
     }
 
     LegacyPlayerKeyStateChange(playerid, newkeys, oldkeys);
     return 1;
 }
 
+// Enable the |left| and or |right| blinkers for |playerid| in |vehicleId|
+SetBlinker(playerid, vehicleId, bool:left, bool:right) {
+    new const blinkerModel = 19294;
+    new const modelId = GetVehicleModel(vehicleId);
+
+    if(IsModelBoat(modelId) || IsModelPlane(modelId)) {
```

Comment:
nit: We already have this information in `VehicleModel`, and we probably want to think about bikes, trailers and helicopters here too. Could we abuse `VehicleModel(modelId)::isNitroInjectionAvailable()`? Otherwise I'd suggest adding a method similar to that with the exact constraints you have in mind.

Reply:
Was not aware of it's existence.
I'm going to use the isNintroInjectionAvailable.
Thank you.